avaframe/avaframeCfg.ini

```diff
@@ -6,6 +6,8 @@
 # Path to avalanche directory
 avalancheDir = data/avaHockeySmoothSmall
 
+# should the results be plotted with a continuous (True) or discrete colormap 
+continuousCmap = False
```

Comment:
Please move this to Flags or move it to the plotting ini file ( I feel the second option is the better one)